web/beacon-app/src/features/teams/routes/TeamsPage.tsx

```diff
@@ -51,14 +58,15 @@ export function TeamsPage() {
             </li>
           </ul>
           <div>
-            {(member?.role === MEMBER_ROLE.OWNER || member?.role == MEMBER_ROLE.ADMIN) && (
-              <Button className="flex items-center gap-1 text-xs" size="small">
+            {hasPermission(USER_PERMISSIONS.COLLABORATORS_ADD) && (
```

Comment:
@daniellemaxwell I had a discussion with @pdeziel about the best way the permission and access control and we choose to use the permission-based control to handle this.